common/network-common/src/main/java/org/apache/spark/network/sasl/SaslClientBootstrap.java

```diff
@@ -88,8 +91,14 @@ public void doBootstrap(TransportClient client, Channel channel) {
           throw new RuntimeException(
             new SaslException("Encryption requests by negotiated non-encrypted connection."));
         }
+
+        if (conf.saslEncryptionAesEnabled()) {
```

Comment:
Naming convention: 
Both Aes and AES are used. Please use the same naming convention. 
Possibly all use AES?